src/internal/interpreter/map-interpreter.test.ts

```diff
@@ -228,6 +228,40 @@ describe('MapInterpreter', () => {
     expect(result.isOk() && result.value).toEqual(144);
   });
 
+  it('should correctly trim and parse path parameters in http call', async () => {
```

Comment:
todo: in the future it would be better to export the function and test it directly

Reply:
You are right, I added the higher-level test because that's how I confirmed the bug reproduction and didn't think to test it at a lower level. I've added a test into `http.test.ts` as well.